package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@onfido/api",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
Assuming we're following [semver](https://semver.org/), I believe we should increase the MINOR version since we're adding new functionality rather than fixing a bug?